build/v1/types.go

```diff
@@ -1430,6 +1433,10 @@ type BuildVolumeSource struct {
 	// configMap represents a ConfigMap that should populate this volume
 	// +optional
 	ConfigMap *corev1.ConfigMapVolumeSource `json:"configMap,omitempty" protobuf:"bytes,3,opt,name=configMap"`
+
+	// CSI represents ephemeral storage that is handled by certain external CSI drivers
```

Comment:
I believe we use lowercase so that the generated OpenAPI doc matchs the yaml/json. Also adding a suggestion to improve the grammar:

```suggestion
	// csi represents ephemeral storage provided by external CSI drivers which support this capability
```